### DISCUSSION TITLE
Overlay profiles

### DISCUSSION THREAD

**Discussion by Erutpur** [runelite/runelite#15523] [Ideas]
If you switch from fullscreen to resizeable into a smaller format make the Boosts Info or Overlays in general with a memory where it was placed at. Right now if you switch from full screen to small, the overlays are messed up and you have to sort them up again. I don't know if that's only me struggling with, but i would appreciate that feature :)

**Comment by Enriath** [comment]
All overlays that aren't in a snap corner are positioned relative to the top left corner. Something like #8515, or something to allow overlays to be positioned relative to other sides/widgets would solve this I think
